app/src/main/java/net/simplifiedcoding/shelounge/Popup/ShowHudService.java

```diff
@@ -0,0 +1,207 @@
+import android.util.DisplayMetrics;
+import android.view.GestureDetector;
+import android.view.Gravity;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.ImageView;
+
+import net.simplifiedcoding.shelounge.R;
+import net.simplifiedcoding.shelounge.ui.WomenSecurity.PentagonActivity;
+
+
+public class ShowHudService extends Service {
+
+    // constants
+    public static final String BASIC_TAG = ShowHudService.class.getName();
+    private GestureDetector gestureDetector;
+    // variables
+    private WindowManager mWindowManager;
+    private Vibrator mVibrator;
+    private WindowManager.LayoutParams mPaperParams;
+    private WindowManager.LayoutParams mRecycleBinParams;
+    private int windowHeight;
+    private int windowWidth;
+
+    // UI
+    private ImageView ivCrumpledPaper;
+    private ImageView ivRecycleBin;
+
+    // get intent methods
+    public static Intent getIntent(Context context) {
+        Intent intent = new Intent(context, ShowHudService.class);
+
+        return intent;
+    }
+
+    // methods
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        mWindowManager = (WindowManager) getSystemService(Service.WINDOW_SERVICE);
+        mVibrator = (Vibrator) getSystemService(Context.VIBRATOR_SERVICE);
+
+        gestureDetector = new GestureDetector(this, new SingleTapConfirm());
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        showHud();
+
+        return START_STICKY;
+    }
+
+    private void showHud() {
+        if (ivCrumpledPaper != null) {
+            mWindowManager.removeView(ivCrumpledPaper);
+            ivCrumpledPaper = null;
+        }
+
+        mPaperParams = new WindowManager.LayoutParams(
+                WindowManager.LayoutParams.WRAP_CONTENT,
+                WindowManager.LayoutParams.WRAP_CONTENT,
+                WindowManager.LayoutParams.TYPE_PHONE,
+                WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE,
+                PixelFormat.TRANSLUCENT);
+
+        DisplayMetrics displaymetrics = new DisplayMetrics();
+        mWindowManager.getDefaultDisplay().getMetrics(displaymetrics);
+        windowHeight = displaymetrics.heightPixels;
+        windowWidth = displaymetrics.widthPixels;
+
+        mPaperParams.gravity = Gravity.TOP | Gravity.RIGHT;
+
+        ivCrumpledPaper = new ImageView(this);
+        ivCrumpledPaper.setImageResource(R.drawable.ic_crumpled_paper);
+
+
+        mPaperParams.x = 0;
+        mPaperParams.y = 100;
+
+        mWindowManager.addView(ivCrumpledPaper, mPaperParams);
+
+        addCrumpledPaperOnTouchListener();
+    }
+
+    private void add() {
+        Intent intent = new Intent(ShowHudService.this, PentagonActivity.class);
+        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+        startActivity(intent);
+    }
+
+    private void addCrumpledPaperOnTouchListener() {
```

Comment:
Not at all understandable the use of this function if the comments are not mentioned